linera-faucet/server/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "linera-faucet-server"
+version = "0.1.0"
```

Comment:
Not sure if this should follow the workspace version (`version.workspace = true`) instead?

Reply:
It definitely should, thank you :)